### DISCUSSION TITLE
Forward Mail without keeping a copy

### DISCUSSION THREAD

**Discussion by yelch** [w2c/ispconfig3_roundcube#133] [Q&A]
Hi. Would it be possible to add an option to forward mails and gibe the users the option NOT to keep a copy?
I've done it until now with a custom rule, but would like to keep the option to the user out of the webmail client...
Any idea?

**Comment by Churro** [comment]
The option would be roundcube-specific but unrelated to ISPConfig, is my understanding correct?

I think you should be able to achieve the same with the managesieve plugin:
https://github.com/roundcube/roundcubemail/blob/master/plugins/managesieve/managesieve.php

**Reply by yelch** [reply]
No I is related to ISPconfig, because I would like to prevent a mixup of settings :I don't see if somebody set a forward-filter with managesieve and the mailbox user does not see if I have prevent the save in ISPC.

I hink easiest would be to set an option for 'Disable local delivery' in the forwarding plugin
![image](https://github.com/w2c/ispconfig3_roundcube/assets/5088023/a1046082-6b9d-47ed-8ecc-613a7e3f4eac)

**Reply by Churro** [reply]
The forwarding plugin implements ISPConfig's "Email Forward" functionality, which is unrelated to the "Send copy to" text input which you see on the "Email Mailbox" screens. I'm not sure if mixing two ISPC functionalities in the forwarding plugin is the right way to go from a functional point of view, although these two seem related at a first glance.

As of now there's no RC plugin to change general mailbox settings like the one you've shown. It might make most sense to add another plugin that allows tweaking these general mailbox settings to disable local delivery, to send a BCC copy to, enable/disable greylisting, etc. If you'd like to contribute this functionality, feel welcome.
